src/content/geothermal/cost.mdx

```diff
@@ -4,6 +4,10 @@ description: The cost to install a geothermal system can range anywhere from $10
 sectionHeader: How Much Does It Cost?
 ---
 
+<img src="/img.png" alt="" />
+
 The cost to install a geothermal system can range anywhere from $10,000 all the way up to $25,000.
 These costs can vary depending on factors such as region, type of system, and how much renovation your house needs.
 Feel free to use our [price calculator](/calculator/) to get an estimate on much it would cost you!
+
+<img src="/images/geothermal_cost.avif" alt="Spacecraft solar power technicians"/>
```

Comment:
Can we have a bit more variety here? Move them around in the article? You can change the size if need be: `<img class="w-52">` (That'll set the width to `208px`/`13rem`).